### DISCUSSION TITLE
We should start a discussion about having more discussions here

### DISCUSSION THREAD

**Discussion by jasikpark** [slackhq/nebula#681] [General]
🧐

**Comment by chris-aeviator** [comment]
rather start answering existing discussions - I see a lot of frustrated users and poor ownership of the repo due to no-answers. nebula seems to not work out of the box for many users as far as i see (and experience myself). Seeing this in a repo makes me not even bother to take the time to file an issue.

**Reply by johnmaguire** [reply]
Hi @chris-aeviator  - I'm sorry to hear that you're frustrated and don't feel it's worth your time to file an issue in the repo.

To provide some context, this discussion was created nearly a year ago, prior to any other threads, as a way of testing out the Discussions feature. Despite our best efforts, it can be difficult to respond to every user report which is part of why we enabled Discussions to try giving the community another forum, outside of the Slack instance, to engage with fellow Nebula users for troubleshooting.

Is there something specific you need assistance with or are you simply looking to vent your frustrations? From reading your comment, it's not clear to me what problems you're running into and I don't see any issues filed under your account. If you're experiencing a bug in Nebula, I suggest filing an issue under the Issues tab.

**Comment by chris-aeviator** [comment]
Ok understand this better. I was having issues and would wish for a better better troubleshooting guides/ documentation on common errors. Therefore it would be benefitial for the team behind nebula to keep track of issues people are facing on day zero operations (https://github.com/slackhq/nebula/issues/834 , https://github.com/slackhq/nebula/issues/836) as well if you offer a community forum this as well. 

Day zero operations are not smooth with nebula. You can see this as an issue filed or tell me to file an issue. I understand that most of these "not able to connect" are hard to debug compared to where people have been able to point out the exact issue.

**Reply by johnmaguire** [reply]
@chris-aeviator Thanks for the feedback. We do have [docs available here](https://nebula.defined.net/docs) including a guide for getting started. The repository is [available here](https://github.com/DefinedNet/nebula-docs/) if you'd like to contribute a troubleshooting guide. We would be happy to see the docs improved.

Thanks for sharing those issues. Indeed, it is difficult to respond to every issue, especially when they do not contain logs or other relevant information for debugging. If you go to the Issue track and click "New issue" you'll see that we actually have a bug report form with information that helps us to debug these issues, but it seems that users are bypassing this form somehow. This is an area in which we'd like to improve, but ultimately we simply do not have the resources to respond to and debug every issue filed, especially when they are described in vague terms without concrete information.

Many users do not run into issues during deployment, but I understand that your day-zero operations were not smooth, and I'm sorry to hear that! Networking is an inherently difficult problem, and there are many ways in which it can go awry - oftentimes due to factors outside of the user's control (e.g. ISP configuration.)

When working to improve Nebula, we find it's best to start with concrete problems users are facing which is where the suggestion to file an issue, containing the exact problems you're running into, came from. 

If there's something you need assistance with at this point, please consider filing an issue using the bug report template and we'll do our best to assist.
